src/components/UserProfile/VolunteeringTimeTab/VolunteeringTimeTab.jsx

```diff
@@ -5,6 +5,8 @@ import { capitalize } from 'lodash';
 import style from '../UserProfileEdit/ToggleSwitch/ToggleSwitch.module.scss';
 import { ENDPOINTS } from 'utils/URL';
 import axios from 'axios';
+import Alert from 'reactstrap/lib/Alert';
```

Comment:
I think we already have this alert part in `UserProfile.js`. I commented out all the <Alert /> things in this file, it did show the alert "Please click on "Save changes" to save the changes you have made."...
<img width="558" alt="image" src="https://user-images.githubusercontent.com/5071040/176321994-4ae2a235-9263-4fe0-81f1-d781026bbfac.png">
I also found your screenshot didn't have that alert. Please go and check other tabs(Teams, Projects,...), if both of them don't have that alert on your dark mode, that would be another bug.
![image](https://user-images.githubusercontent.com/5071040/176322136-a8ae8697-584b-40f8-a37f-87a7b88d36a7.png)

![image](https://user-images.githubusercontent.com/5071040/176322069-9302981e-a78a-4903-b087-028350fb7571.png)